setup.cfg

```diff
@@ -83,7 +83,7 @@ docs =
 
 dev =
     bandit
-    black == 19.10b0
+    black == 22.3.0
```

Comment:
pin or just leave free to install latest?

Reply:
The style is coupled to the `black` version so I would leave it pinned. Id rather have all the style changes in one PR

Reply:
Probably the same for isort and pylint. I need to fix an unrelated piece of code to get this PR passing because of an update to pylint.

Reply:
Sounds good